app/assets/stylesheets/likes.scss

```diff
@@ -1,3 +1,41 @@
 // Place all the styles related to the Likes controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+.like_button, .like_count
+{
+  display:inline;
+}
```

Comment:
Our current style is:

```css
.like-button, .like-count {
  display: inline;
}
```

So:

* Hyphens for class names
* First curly brace on the same line as the selectors